msal/src/Microsoft.Identity.Client/UIParent.cs

```diff
@@ -116,20 +118,24 @@ public static bool IsSystemWebviewAvailable()
 
             int counter = 0;
 
-            ApplicationInfo applicationInfo = Application.Context.PackageManager.GetApplicationInfo(_chromePackages[0], 0);
-
-            for (int i = 0; i < _chromePackages.Length; i++)
+            ApplicationInfo applicationInfo = Application.Context.PackageManager.GetApplicationInfo(_customTabPackages[0], 0);
```

Comment:
>0 [](start = 119, length = 1)

if chrome has special meaning, I would pull it out in a separate constant. You can still keep it in _customTabPacakges.